manage/slas/how-do-they-work.rst

```diff
@@ -15,6 +15,15 @@ SLAs as their calculations may not work as you intended them originally.
    depending on your instance size and affected tickets. You may want to perform
    those changes outside of your business hours.
 
+.. tip::
+
+   Communication type articles are not enough for your SLA?
+   You can also count notes towards your SLA, `our console documentation`_
```

Comment:
"You can also configure Zammad to allow public **notes** as fullfilments for SLA requirements, ..." instead of "You can also count notes towards your SLA,".